TYPO3.Neos/Classes/TYPO3/Neos/Utility/BackendAssetsUtility.php

```diff
@@ -39,7 +39,7 @@ public function injectSettings(array $settings)
      */
     public function shouldLoadMinifiedJavascript()
     {
-        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : $this->settings['userInterface']['loadMinifiedJavascript'];
+        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : true;
```

Comment:
don't really agree, it was added to make it easier and avoid that problem in a easy way.. why remove it again? throwing an error is for a misspelled configuration option is pretty weird IMO.. and with this change it where it defaults to true if not set you cannot check if it's not set, only check if it's spelled the wrong way..